test/NuGet.Clients.FuncTests/NuGet.CommandLine.FuncTest/Commands/PushCommandTest.cs

```diff
@@ -18,7 +18,7 @@ public class PushCommandTest
         private const string MESSAGE_PACKAGE_PUSHED = "Your package was pushed.";
         private const string TEST_PACKAGE_SHOULD_NOT_PUSH = "The package should not have been pushed";
         private const string TEST_PACKAGE_SHOULD_PUSH = "The package should have been pushed";
-        private const string ADVERTISE_CONTINUE_OPTION = "See help for push option to automatically skip duplicates.";
+        private const string ADVERTISE_SKIPDUPLICATE_OPTION = "To skip already published packages, use <command> -skipduplicate"; //PushCommandSkipDuplicateAdvertiseNuGetExe
```

Comment:
There's an internals visible to from nuget.commandline to this assembly. 
You can just use the resource directly.

Reply:
This is the Protocol assembly